ISSUE_TEMPLATE.md

```diff
@@ -1,16 +1,13 @@
-Please go to Stack Overflow for help and support.
+Please go to Stack Overflow for help and support:
+
 http://stackoverflow.com/questions/tagged/tensorflow
 
 If you open a GitHub issue, here is our policy:
 
 1. It must be a bug or a feature request.
 2. The form below must be filled out.
 
-**Here's why we have that policy**: TensorFlow developers respond to issues.
-We want to focus on work that benefits the whole community, e.g., fixing
-bugs and adding features. Support only helps individuals. GitHub also notifies
-thousands of people when issues are filed. We want them to see you communicating
-an interesting problem, rather than being redirected to Stack Overflow.
+**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.
```

Comment:
@nealwu In my previous PR, I added the line breaks to conform to the 80-char line width limit. I thought those are okay. Maybe there is some reason I didn't realize.